daemon/vpn/nordlynx/libtelio/libtelio.go

```diff
@@ -183,15 +218,8 @@ func New(prod bool, eventPath string, fwmark uint32,
 	if err != nil {
 		log.Println(internal.ErrorPrefix, "Failed to get telio config:", err)
 
-		defaultTelioConfig := &telioFeatures{}
-		defaultTelioConfig.Lana = &lanaConfig{
-			Prod:      prod,
-			EventPath: eventPath,
-		}
-		defaultTelioConfig.Direct = &directConfig{}
-		defaultTelioConfig.Nurse = &nurseConfig{
-			Fingerprint: deviceID,
-		}
+		defaultTelioConfig := newEmptyTelioConfig()
```

Comment:
It is not the same behavior as before. With this changes the direct connections and events will not work anymore.

Reply:
also shouldn't the new values be added with some default ones?

Reply:
1st comment - you are right, 2nd - there was discussion with libtelio and agreement with them is that default values are set/handled by libtelio - consumers from outside should specify empty value (at any level) for libtelio to know to go with defaults.